src/common/storage/storage-wrapper.js

```diff
@@ -59,15 +63,16 @@ export default class StorageWrapper {
   /**
    * Gets an item from the local storage.
    * @param {string} key - The item key.
+   * @param {Object} storage - The storage object to use.
    * @static
    * @private
    * @returns {any} - The item value.
    */
-  static getItem(key: string): any {
+  static getItem(key: string, storage: Object = localStorage): any {
```

Comment:
I think a better solution would be to wrap
StorageWrapper.setItem in a function on the manager side, if you want to avoid passing the 2nd parameter every time.
I don't think the wrapper should know what kind of storage objects are available to it. just their API.

Reply:
hmm not sure i understand how the wrapper function on the manager side should look like and how it will help avoiding passing the storage as a parameter. either way eventually if the setItem (or getItem) is generic, then it has to receive the storage as well, no?
i thought it makes sense to have localStorage as the default, which means i don't need to change it on local storage manager side, but only on session storage manager side.